src/plexil-adapter/OwInterface.cpp

```diff
@@ -153,20 +154,35 @@ const map<string, string> ArmFaults
   { "/faults/scoop_yaw_torque_sensor_failure", "Scoop Yaw Torque Sensor" }
 };
 
+const map<string, string> PowerFaults
+{
+  // Param name -> human-readable
+  { "/faults/low_state_of_charge_power_failure", "State Of Charge" },
+  { "/faults/instantaneous_capacity_loss_power_failure", "State of Charge" },
+  { "/faults/thermal_power_failure", "Thermal Power" }
+};
+
+const std::map<string, string> unionTwoFaultMaps(std::map<std::string, std::string> map1, std::map<std::string, std::string> map2)
```

Comment:
It's cleaner to omit the "std::" in front of map and string.  
To avoid unneeded copying in args, the source maps should be passed const&.
Also, shorten this line by stacking the two input args -- much more readable.
There may be a cleaner approach to all this at a higher level but I don't have suggestions at the moment.

Reply:
done!